Fix bug type in EnergyMarketCodeType

camille-bouvy-frequenz opened the pull request:
The input parameter `code_type` of `DeliveryArea` was the protobuf type `delivery_area_pb2.EnergyMarketCodeType.ValueType` instead of the python wrapper type `EnergyMarketCodeType`.

matthias-wende-frequenz reviewed (changes requested):
Looks good. Can you please split the changes in two commits and add the summery in the PR also to the corresponding commit title?

Commit:
Fix bug in EnergyMarketCodeType type

The input parameter code_type of DeliveryArea was the protobuf type
delivery_area_pb2.EnergyMarketCodeType.ValueType instead of the python
wrapper type EnergyMarketCodeType.

Signed-off-by: camille-bouvy-frequenz <camille.bouvy@frequenz.com>

Commit:
Update release notes

Signed-off-by: camille-bouvy-frequenz <camille.bouvy@frequenz.com>

ghost commented:
@matthias-wende-frequenz updated 👍
